devs/docs/tests.rst

```diff
@@ -10,6 +10,28 @@ Run the all of `doctests`_::
 
     $ ./gradlew itest
 
+For running itest on Windows, `WSL needs to be installed`_. If the downloaded
+Linux distro comes without Java, you can install it by running the following
+commands::
+
+    $ sudo add-apt-repository ppa:openjdk-r/ppa
+    $ sudo apt-get update
+    $ sudo apt install openjdk-11-jdk.
+
+Then install python virtual environment by running::
+
+    $ sudo apt-get install python3-venv
+
+To support symlinks, `enable Developer Mode`_ on the "For developers" settings
+page and run::
+
+    $ git config --global core.symlinks true
```

Comment:
While this part of the documentation can stay here, I wonder if we should additionally add a `.. NOTE::` section about "development on Windows" near the `git clone` command within the "Basic setup" document [1].

Regarding this, I assume that the "developer mode" has to be activated before actually cloning the repository. Is this true?

[1] https://github.com/crate/crate/blob/4.5.0/devs/docs/basics.rst#prerequisites

Reply:
Or, alternatively, let's keep this fenced to this document but add a respective note here that ``git clone`` would have to be invoked again after adjusting the WSL environment.

Reply:
minor nit: GitHub RST rendering doesn't support admonitions (e.g., `NOTE`). check out https://github.com/crate/crate-docs/blob/main/style/hypertext.rst for an example of how I work around this (the note block starting with "Link text noun phrases should not be self-referential.")

Reply:
Oh, true. Thank you for the heads-up, Naomi!

Reply:
Good point, thanks. Currently checking whether symlink created proactively after enabling dev mode.

Reply:
So, just enabling dev mode and setting `symlinks = true` is not enough. I wrote a bit different approach to checkout the file (fetch + checkout) as for cloning one needs to remove directory and re-clone again (If I'm not missing something)

Reply:
That was kind of the thing I was expecting. When symlinks are needed to reflect the repository contents, support for them should be enabled _before_ cloning the repository.

Reply:
I found out another way - switching branches also helps in that regard. Added this as a short note. 
I can mention enabling Dev mode in prerequisites as well for double safety - wdyt?

Reply:
Thank you. I believe it is good to go.